linode/helper/retry.go

```diff
@@ -69,6 +48,12 @@ func ImageUpload500Retry() func(response *resty.Response, err error) bool {
 		}
 
 		// Check whether the string matches
-		return ImageUpload.MatchString(requestURL.Path)
+		return pathPattern.MatchString(requestURL.Path)
 	}
 }
+
+func ApplyAllConditions(client *linodego.Client) {
```

Comment:
nit: Probably more clear to add `retry` in the function names, i.e. `GenericRetryCondition`, `ApplyAllRetryConditions`?

Reply:
Thanks for the suggestion! I just added it in the latest commit.